utils/mail/templateWaitlist.ts

```diff
@@ -0,0 +1 @@
+// email ?
```

Comment:
+ template for email would be here;

i think i'd pass in params for the class + its related data to be displayed in the email? i'm thinking the email can say something like

"X class currently has spots available" + provide some info (date, time of class etc)
-> from there, the email can have a button that links directly to the browse classes page? and button can say "Register Now" or something
-> we could also add another button that says "Remove from Waitlist" and link it to the MyClasses page?

@bekahma @vedantdsn @MatoPlus what do you think ^^ ; if you have any ideas for how you want the email to look like let me know! otherwise, i don't mind coding up the draft and we can make alterations after